src/pages/applicants/project-grants/apply.tsx

```diff
@@ -15,37 +12,26 @@ const ProjectGrantsApply: NextPage = () => {
       />
 
       <Box bg='white' position='relative' py={{ md: 12 }} px={{ md: 24, lg: 32, xl: 72 }}>
-        <Stack>
-          <Stack mb={8}>
-            <section id='description'>
-              <PageSubheading mb={8} textAlign='center'>
-                Apply to Project grants
-              </PageSubheading>
-
-              <PageText>
-                This webform collects basic information about you and your project. You&apos;ll also
-                need to download and fill out the full Project Grants application form, and upload
-                it with your submission. We take this approach because we want you to take your time
-                and be thoughtful in your responses, not be limited by formatting, and not have the
-                stress of trusting an online form with hours&apos; worth of work. Don&apos;t feel
-                constrained by the format, but make sure you answer all of the application questions
-                clearly.
-              </PageText>
-            </section>
-          </Stack>
+        <section id='description'>
+          <PageSubheading mb={8} textAlign='center'>
+            Apply to Project grants
+          </PageSubheading>
 
-          <Stack>
-            <section id='download-application'>
-              <Center>
-                <ButtonLink
-                  label='Download Application'
-                  link={DOWNLOAD_APPLICATION_URL}
-                  width='268px'
-                />
-              </Center>
-            </section>
-          </Stack>
-        </Stack>
+          <PageText>
+            This webform collects information about you and your project. Please use the spaces
+            below to answer the following questions thoughtfully and thoroughly. The information
+            you provide now is what we&apos;ll use to determine whether to award a grant. If you have
+            any questions, please visit the{' '}
+            <Link
```

Comment:
Chakra's `<Link>` components that don't use an external href should be wrapped with [Next's Link component](https://nextjs.org/docs/api-reference/next/link) for client-side transitions, e.g.: in `ApplicantsSidebar.tsx` we have

```
<NextLink href={href}>
  <Link href={href} _hover={{ textDecoration: 'none' }}>
    <PageText fontWeight={400} lineHeight='28px' color='brand.orange.100'>
      {text}
    </PageText>
  </Link>
</NextLink>
 ```
 
 Notice that both components are called `Link`, that's why the one from Next is usually imported with a different name, to avoind conflicts, like `import NextLink from 'next/link';`

Reply:
Strange. We can leave just the <Link> for now I think to unblock this and got it merged.